### PR TITLE
Updated PR template based on feedback

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Fix: Updating PR template. Not a significant contribution

For significant contributions please make sure you have completed the following items:

- [ ] Design discussion issue #
- [ ] Changes in public surface reviewed
- [ ] CHANGELOG.md updated
- [ ] Ran atleast all unit tests locally
- [ ] Ran functional tests locally if making major changes.
- [ ] The PR will trigger build, unit test, and functional tests automatically. If you are non-Microsoft employee, please
mention any committers to help trigger build.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 
-- [ ] Design discussion issue #
 - [ ] Changes in public surface reviewed
-- [ ] CHANGELOG.md updated
-- [ ] Ran atleast all unit tests locally
-- [ ] Ran functional tests locally if making major changes.
-- [ ] The PR will trigger build, unit test, and functional tests automatically. If you are non-Microsoft employee, please
-mention any committers to help trigger build.
+- [ ] I ran [functional tests](https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/develop/CONTRIBUTING.md#functional-tests) locally.
+- [ ] The PR will trigger build, unit test, and functional tests automatically. If you are non-Microsoft employee, please mention any committers to help trigger build.
```
Comment (cijothomas) [inline .github/PULL_REQUEST_TEMPLATE.md]: If you are non-Microsoft employee or if you are submitting PR from a forked repository

Comment (SergeyKanzhelev) [reply .github/PULL_REQUEST_TEMPLATE.md]: fixed

**Comment by SergeyKanzhelev** [comment]
@cijothomas said this specific test case is flacky. Merging now. 

@cijothomas is there an issue we are tracking to fix this unit test?

**Comment by cijothomas** [comment]
@SergeyKanzhelev i meant this test failure is likely one-time and should work on retry. Unless this keeps occuring, no item to track.

**Pull request merged**